docs/sql-ref-syntax-dml-insert-into.md

```diff
@@ -198,6 +198,21 @@ SELECT * FROM students;
 +-------------+-------------------------+----------+
 ```
 
+#### Insert Using a Type Constructed Partition Column Value
```

Comment:
`Insert Using a Typed Date Literal for a Partition Column Value`?

Reply:
Done